pkg/apis/datadoghq/v1alpha1/datadogagentdeployment_types.go

```diff
@@ -171,6 +175,22 @@ type APMSpec struct {
 	// ref: https://github.com/DataDog/docker-dd-agent#tracing-from-the-host
 	// +optional
 	Enabled *bool `json:"enabled,omitempty"`
+	// Number of port to expose on the host.
+	// If specified, this must be a valid port number, 0 < x < 65536.
+	// If HostNetwork is specified, this must match ContainerPort.
+	// Most containers do not need this.
+	// +optional
+	HostPort *int32 `json:"hostPort,omitempty"`
+
+	// The Datadog Agent supports many environment variables
+	// ref: https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/agent#environment-variables
```

Comment:
looks like the readme in that [path](https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/agent#environment-variables) has changed

Reply:
new path: https://docs.datadoghq.com/agent/docker/?tab=standard#environment-variables